lib/build.gradle

```diff
@@ -1,15 +1,20 @@
 apply plugin: 'com.android.library'
+apply plugin: "com.gladed.androidgitversion"
 
-apply plugin: 'com.gladed.androidgitversion'
+androidGitVersion {
+    codeFormat "MMNNPP"
+    format '%tag%'
+    tagPattern(/^[0-9]+.*/) // Tag names should follow the pattern MM.NN.PP
+}
 
 android {
-    compileSdkVersion project.ext.compileSdkVersion
+    compileSdkVersion 30
 
     defaultConfig {
-        minSdkVersion project.ext.minSdkVersion
-        targetSdkVersion project.ext.targetSdkVersion
-        versionCode 1
+        minSdkVersion 21
```

Comment:
What do you think about doesnt couple example app with the lib? @adriangl

Reply:
I was talking only about lib gradle file. Its a bit awkward need to look at the example app to get what `minSdk` version has the library, do you understand why I mean?

I'm OK with has `versionCode` to 1 at app's gradle.

Reply:
Sorry for the delay. Yeah, we can leave the versions decoupled ;)